roadmap.md

```diff
@@ -8,26 +8,40 @@ Absa's forked version was brought back under Hyperledger umbrella under name `ar
 The further development of the library can be split into a few phases, roughly marking important 
 architectural milestones.
 
-# Phase 0
+# Phase 0 - Done ✅
 - We could call the phase from Absa's fork up until return to Hyperledger as Phase 0, the most of the 
 code cleanup and code restructuring happened. We have
-- deleted legacy code, 
-- migrated CI to Github Actions (building docker, ios, android artifacts)
-- changed library testing approach (favoring integration testing in language wrappers, removing 
+- ✅ deleted legacy code, 
+- ✅ migrated CI to Github Actions (building docker, ios, android artifacts)
+- ✅ changed library testing approach (favoring integration testing in language wrappers, removing 
   encrypted mock inputs on rust-level unit testing)
-- thinned language wrappers (wrappers should use functions to access data, rather trying to map out 
+- ✅ thinned language wrappers (wrappers should use functions to access data, rather trying to map out 
   Rust data structures).
 
-# Phase 1
-This phase is all about decoupling parts of the library into independent modules. We are currently 
-targeting extraction of 2 independent reusable modules:
-- mediator agent client
-- Aries protocol state machines
-Once that is done, `aries-vcx` will essentially become opinionated C-Callable glue of independent rust 
-crates, rather than single monolithic opinionated library. Additionally `aries-vcx` itself should 
-become crate consumable from other Rust projects.
+# Phase 1 - Done ✅
+This phase is all about decoupling parts of the library into independent modules. We have decoupled 
+the library into 3 pieces.
+- ✅ `mediator agent client` - client for talking to a compatible agencies - the only open source 
+  implementation available is [vcxagencynode](https://github.com/AbsaOSS/vcxagencynode).
+- ✅ `aries_vcx` - the "glue" between Aries state machines, `libindy` and mediator agent.  
+- ✅ `libvcx` - adds memory management and C bindings on top of `aries_vcx` - making it consumable
+   on Android, iOS and any programming language.
+
+# Phase 2 - In progress 🚧
+- 🚧 Concising `aries-vcx` crate API and start publishing on crates.io
```

Comment:
If I were to be nitpicky, I would say use either simple or continuous form but not both